File: benchmark/corpus/05_overflow/b_square_input.py
# Squares a number from input; 4000000001 squared is past 64 bits.
import sys


def square(n):
    return n * n  # DANGER


line = sys.stdin.readline().strip()
try:
    n = int(line)
except ValueError:
    print("not a number: " + line)
else:
    print(square(n))

File: benchmark/corpus/05_overflow/b_square_input.js
// Squares a number from input; 4000000001 squared is past 64 bits.
function square(n) {
  return n * n; // DANGER
}

const line = (await new Response(Deno.stdin.readable).text()).trim();
const n = Number(line);
if (Number.isNaN(n)) console.log("not a number: " + line);
else console.log(square(n));
